deid/dicom/header.py

```diff
@@ -46,7 +46,7 @@
 import dateutil.parser
```

Comment:
Looks like the DS STORE file snuck in? You can add this to the gitignore and remove the file or when you add files don't add entire folders at once.